ApiFirst/AppServices/Services/CustomerAppService.cs

```diff
@@ -21,14 +22,15 @@ public CustomerAppServices(ICustomerService customerServices, IMapper mapper)
 
         public CustomerResponseDTO GetByCpf(string cpf)
         {
+            cpf = FormatStringExtension.FormatString(cpf);
             var result = _customerServices.GetByCpf(cpf);
```

Comment:
A chamada do extension method está errada. O método é criado justamente para poder ser utilizado em qualquer string. Assim, ela deve ser chamada na variável
```suggestion
        {
            cpf = cpf.FormatString());
            var result = _customerServices.GetByCpf(cpf);
```